vllm/worker/neuron_model_runner.py

```diff
@@ -121,13 +121,13 @@ def _prepare_prompt(
         max_seq_len = max(seq_lens)
         assert max_seq_len > 0
```

Comment:
Probably can move these 2 lines into the padding function as well?

Reply:
We should keep this as it avoids recomputing the `max_seq_len` for each invocation of `make_tensor_with_pad`.

Reply:
nvm I misread it. It is still used by the following calls.